dev/archery/archery/integration/datagen.py

```diff
@@ -939,6 +961,35 @@ def generate_column(self, size, name=None):
         return StructColumn(name, size, is_valid, field_values)
 
 
+class RunEndEncodedField(Field):
+
+    def __init__(self, name, run_ends_bitwidth, values_field, *, nullable=True,
+                 metadata=None):
+        super().__init__(name, nullable=nullable,
+                         metadata=metadata)
+        self.run_ends_field = RunEndsField(
+            'run_ends', run_ends_bitwidth, nullable=False)
+        self.values_field = values_field
+
+    def _get_type(self):
+        return OrderedDict([
+            ('name', 'runendencoded')
```

Comment:
```suggestion
            ('name', 'run_end_encoded')
```

Reply:
So i was following the existing pattern we have for the integration tests. The names for the types all don't have spaces in them for example `fixedsizebinary` and `fixedsizelist`. I'd rather keep the consistency and leave it as `runendencoded`. Thoughts?

Reply:
Oh, I see. Well, let's keep `runendencoded` then :-)